### PR TITLE
Update docs for csi topology ga

### PR DESCRIPTION
**Pull request by msau42**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by msau42** [comment]
/hold
until https://github.com/kubernetes/kubernetes/pull/83474 merges

**Comment by jsafrane** [comment]
LGTM from tech side

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ different Kubernetes components.
 | `CSIMigrationGCE` | `false` | Alpha | 1.14 | |
 | `CSIMigrationOpenStack` | `false` | Alpha | 1.14 | |
 | `CSINodeInfo` | `false` | Alpha | 1.12 | 1.13 |
-| `CSINodeInfo` | `true` | Beta | 1.14 | |
+| `CSINodeInfo` | `true` | Beta | 1.14 | 1.16 |
+| `CSINodeInfo` | `true` | GA | 1.17 | |
```
Comment (jsafrane) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Should be the whole feature moved to the "Feature gates for graduated or deprecated features" table below?

Comment (msau42) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: done

**Comment by daminisatya** [comment]
Thank you @sftim @jsafrane for the review 

/lgtm 
/approve

**Pull request merged**